src/pydicom/data/urls.json

```diff
@@ -20,8 +20,14 @@
     "JPEG2000_UNC.dcm": "https://github.com/pydicom/pydicom-data/raw/39a2eb31815eec435dc26c322c27aec5cfcbddb6/data/JPEG2000_UNC.dcm",
     "JPGLosslessP14SV1_1s_1f_8b.dcm": "https://github.com/pydicom/pydicom-data/raw/39a2eb31815eec435dc26c322c27aec5cfcbddb6/data/JPGLosslessP14SV1_1s_1f_8b.dcm",
     "liver.dcm": "https://github.com/pydicom/pydicom-data/raw/39a2eb31815eec435dc26c322c27aec5cfcbddb6/data/liver.dcm",
+    "liver_rle.dcm": "https://github.com/CPBridge/pydicom-data/raw/ace9f200cbf3773f30d196a3924b77f81dfebe96/data_store/data/liver_rle.dcm",
+    "liver_deflate.dcm": "https://github.com/CPBridge/pydicom-data/raw/ace9f200cbf3773f30d196a3924b77f81dfebe96/data_store/data/liver_deflate.dcm",
+    "liver_j2k.dcm": "https://github.com/CPBridge/pydicom-data/raw/ace9f200cbf3773f30d196a3924b77f81dfebe96/data_store/data/liver_j2k.dcm",
     "liver_expb.dcm": "https://github.com/pydicom/pydicom-data/raw/39a2eb31815eec435dc26c322c27aec5cfcbddb6/data/liver_expb.dcm",
-    "liver_nonbyte_aligned.dcm": "https://github.com/pydicom/pydicom-data/raw/8da482f208401d63cd63f3f4efc41b6856ef36c7/data_store/data/liver_nonbyte_aligned.dcm",
+    "liver_nonbyte_aligned.dcm": "https://github.com/CPBridge/pydicom-data/raw/ace9f200cbf3773f30d196a3924b77f81dfebe96/data_store/data/liver_nonbyte_aligned.dcm",
+    "liver_nonbyte_aligned_deflate.dcm": "https://github.com/CPBridge/pydicom-data/raw/ace9f200cbf3773f30d196a3924b77f81dfebe96/data_store/data/liver_nonbyte_aligned_deflate.dcm",
+    "liver_nonbyte_aligned_rle.dcm": "https://github.com/CPBridge/pydicom-data/raw/ace9f200cbf3773f30d196a3924b77f81dfebe96/data_store/data/liver_nonbyte_aligned_rle.dcm",
+    "liver_nonbyte_aligned_j2k.dcm": "https://github.com/CPBridge/pydicom-data/raw/ace9f200cbf3773f30d196a3924b77f81dfebe96/data_store/data/liver_nonbyte_aligned_j2k.dcm",
```

Comment:
Can you PR these to pydicom-data please?

Reply:
I did that earlier, see https://github.com/pydicom/pydicom-data/pull/16

Then we need to remember to change the urls here

Reply:
Sorry, apparently at some point I must've turned the notifications level down.